### ISSUE TITLE
newman globals exported with --export-globals cannot be loaded into postman

### ISSUE THREAD

**Issue by connorsadlervelo** [postmanlabs/newman#1562]
newman: 3.9.4
postman: Version 6.1.3 (6.1.3)
Macbook Pro 16 GB , Sierra 10.12.6

I am running newman with "--export-globals" to produce a .json file containing global variable values.
I then try to Import that file into Postman but get an error: 
Error while importing: format not recognized.

After some digging (comparing with a globals file exported from Postman) I found the "name" field is missing in the globals file created by newman.
If I add in the following line near the bottom of the file created by newman, it imports into Postman OK:
   "name": "XXX",

Should the file formats be identical?

Regards,

Connor

**Script to run newman:**
newman run "CFS export globals test.postman_collection.json" --bail failure -r cli --export-globals "CFS export globals test - exported globals from newman.json"

**Collection: CFS export globals test.postman_collection.json**
```
{
	"info": {
		"_postman_id": "38c0e9de-af67-4e1a-9e43-fec47fa5b868",
		"name": "CFS export globals test",
		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
	},
	"item": [
		{
			"name": "Change Global 1",
			"event": [
				{
					"listen": "test",
					"script": {
						"id": "a666463b-75d4-4d0b-ab9a-c0f766516d54",
						"type": "text/javascript",
						"exec": [
							"postman.setGlobalVariable(\"myGlobal1\", \"hello\")",
							"postman.setGlobalVariable(\"myGlobal2\", \"world\")"
						]
					}
				}
			],
			"request": {
				"method": "GET",
				"header": [],
				"body": {},
				"url": {
					"raw": "https://postman-echo.com/delay/0",
					"protocol": "https",
					"host": [
						"postman-echo",
						"com"
					],
					"path": [
						"delay",
						"0"
					]
				}
			},
			"response": []
		}
	]
}

```

**Comment by connorsadlervelo** [comment]
For completeness:

**File exported from newman:**
```
{
  "id": "d51618d9-0c34-407e-8d3f-bba720dddaa2",
  "values": [
    {
      "type": "any",
      "value": "hello",
      "key": "myGlobal1"
    },
    {
      "type": "any",
      "value": "world",
      "key": "myGlobal2"
    }
  ],
  "_postman_variable_scope": "globals",
  "_postman_exported_at": "2018-06-01T16:36:21.593Z",
  "_postman_exported_using": "Newman/3.9.4"
}
```

**File exported from Postman:**

```
{
  "id": "eca45624-0141-4dd1-8329-7f2bcb5e1678",
  "values": [
    {
      "key": "myGlobal1",
      "value": "hello",
      "enabled": true,
      "type": "text"
    },
    {
      "key": "myGlobal2",
      "value": "world",
      "enabled": true,
      "type": "text"
    }
  ],
  "name": "My Workspace Globals",
  "_postman_variable_scope": "globals",
  "_postman_exported_at": "2018-06-01T16:37:26.303Z",
  "_postman_exported_using": "Postman/6.1.3"
}
```

**Comment by kunagpal** [comment]
@connorsadlervelo This is happening because the exported variables file will not have a name when there are no variables to begin with, as is the case here. Thanks for reporting this, I've created a patch here: #1563 😄

**Comment by kunagpal** [comment]
@connorsadlervelo This has been fixed in Newman v3.10.0 😄
